### PR TITLE
feat: seax-bulk send agent wiki

### PR DESCRIPTION
**Pull request by itIsJim**

### CODE REVIEW

File: content/en/seax/seax_messaging/seax-agent-bulk-send.md
[the remark this answers is not in the document]
Comment (ChiaHsiangKuo) [reply content/en/seax/seax_messaging/seax-agent-bulk-send.md:1]: For ```Senders```, I think we don't need to ``` Ensure the selected phone number has the necessary channels enabled.```
We should have already disabled users to select numbers that do not have the corresponding capability.

[the remark this answers is not in the document]
Comment (itIsJim) [reply content/en/seax/seax_messaging/seax-agent-bulk-send.md:1]: will fix

[the remark this answers is not in the document]
Comment (ChiaHsiangKuo) [reply content/en/seax/seax_messaging/seax-agent-bulk-send.md:1]: ```Update Inbound Agent``` is not just for SMS, but Calls as well.

**Review by ChiaHsiangKuo — APPROVED** [review]
Checked and approved.

**Pull request merged**